Master 1.10.2 dev

knoxhack opened the pull request:
Updated latest changes from dev version.

knoxhack committed:
Enum shit

knoxhack committed:
enums

knoxhack committed:
Added recipe to Outfit

knoxhack committed:
Added Outfit Recipe

knoxhack committed:
Added back the Male Farmhand+Farmland biome spawning

knoxhack committed:
render fixes

knoxhack committed:
tile blocks variants + render fixes

knoxhack committed:
implemented enum for fancy tile blocks + fixes

knoxhack committed:
Added Large,Small,and Tiny tiles + more efficient enums

knoxhack committed:
starting mobs rewrite for mods compat

knoxhack committed:
clean up

knoxhack committed:
chinkens rewrite + cow mob drops

knoxhack committed:
added drop items textures + more mob rewrites

knoxhack committed:
Moved to Version 1.0 RC1

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/MinecraftModDevelopment/BetterAgriculture?pullRequest=31) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/MinecraftModDevelopment/BetterAgriculture?pullRequest=31) before we can accept your contribution.<br/>

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/MinecraftModDevelopment/BetterAgriculture?pullRequest=31) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/MinecraftModDevelopment/BetterAgriculture?pullRequest=31) before we can accept your contribution.<br/>

knoxhack committed:
fixed cows droping slate

knoxhack committed:
started integration with MMDLib

Added:
Stairs,Slabs,Walls,Ores
started integration to the new config

knoxhack committed:
build 2

chages material to ROCK.
added a bunch of JSONs for the all the TinyTile variant blocks.
